### ISSUE TITLE
Money as Double?

### ISSUE THREAD

**Issue by DiegoNolan** [WraithM/lendingclub#1]
Why is money not fixed point? Haskell has Rational.

**Comment by wraithm** [comment]
The reason why I did this is that the Lending Club API uses floating point,
at least superficially (eg.
https://www.lendingclub.com/developers/summary.action). You caught me. I
was lazy.

You're absolutely right that it makes more sense to use fixed point
instead. Something like `scientific` might be a good candidate.

What do you think `Integer`, `scientific`, something else? I'm very open to
other suggestions.

**Comment by dmjio** [comment]
Scientific or Decimal, probably

**Comment by DiegoNolan** [comment]
I'm wondering if it is necessary though. If we are using application/json I don't know if it matters since in javascript all numbers are represented as doubles. I guess when we end of doing arithmetic operations on the results we get from the API it may matter.

**Comment by wraithm** [comment]
Yeah, you're exactly where I was thinking when I wrote this thing. You're
exactly right that lending club is using doubles for the json, and that's
why I used doubles. Again, you're right that you get precision issues when
you do arithmetic on that data using this library.

I don't know anything about `Decimal`. My favorite would probably be using
`Integer`.
